### PR TITLE
Custom items 09/06

### LINKED ISSUE #4783: Custom Item Spawning in Cargo
Spark Gloves spawn in cargo at random. I assume that's a bug because there was no exclusion set-up for it to not spawn in cargo nabbing things under clothing/gloves.
spark gloves

### PR DESCRIPTION
**Pull request by Alberyk**
Adds the following items:

Custom Martian Raider T-Shirt - Caiden Moyers - tylaaaar
Senior Alchemist Robes - Eri Akhandi - paradoxspace
Solarian Alliance Military ID - Kalren Halstere - brutishcrab51

Config
[custom_items.txt](https://github.com/Aurorastation/Aurora.3/files/2087484/custom_items.txt)

-fixes #4783

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/obj/custom_items/eri_robes.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
eri_robes||eri_robes|Created
eri_robes_su||eri_robes_su|Created
eri_robes_su[1]||![eri_robes_su[1]](http://images.aurorastation.org/images/82867dd7f49d937b5f81613b9a9cc898.png)|Created
eri_robes_su[2]||![eri_robes_su[2]](http://images.aurorastation.org/images/47a9700b6838c0c9b91ad3ed26827218.png)|Created
eri_robes_su[3]||![eri_robes_su[3]](http://images.aurorastation.org/images/5e9b7249dfe62621480d4cd3ae26fe74.png)|Created
</details>
<details><summary>icons/obj/custom_items/halstere_clothing.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
halstere_card||halstere_card|Created
halstere_card_open||halstere_card_open|Created
</details>
<details><summary>icons/obj/custom_items/moyers_shirt.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
moyers_shirt||moyers_shirt|Created
moyers_shirt_un||moyers_shirt_un|Created
moyers_shirt_un[1]||![moyers_shirt_un[1]](http://images.aurorastation.org/images/47c528ac9f86932765c7cdd72fcd2215.png)|Created
moyers_shirt_un[2]||![moyers_shirt_un[2]](http://images.aurorastation.org/images/12281d1db8e96c207f24be5e66f6d178.png)|Created
moyers_shirt_un[3]||![moyers_shirt_un[3]](http://images.aurorastation.org/images/6c45ab3ad8ef826e4339dc72a94005c7.png)|Created
</details>

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -2265,4 +2265,46 @@ obj/item/clothing/suit/storage/hooded/fluff/make_poncho //Raincoat Poncho - M.A.
...
 	new /obj/item/fluff/yarn/green(src)
 	new /obj/item/fluff/yarn/purple(src)
-	new /obj/item/fluff/yarn/yellow(src)
\ No newline at end of file
+	new /obj/item/fluff/yarn/yellow(src)
+
+
+/obj/item/clothing/under/fluff/moyers_shirt //Custom Martian Raider T-Shirt - Caiden Moyers - tylaaaar
+	name = "custom martian raider t-shirt"
+	desc = "A Martian Raider Spaceball T-shirt with the name \"MOYERS\" Written on the back in plain white text."
```
Comment (Lohikar) [inline code/modules/customitems/item_defines.dm]: `Written` should not be capitalized here.

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -2265,4 +2265,46 @@ obj/item/clothing/suit/storage/hooded/fluff/make_poncho //Raincoat Poncho - M.A.
...
+	name = "solarian alliance military ID"
+	desc = "A green and white military identification card, with an Alliance of Sovereign Solarian Nations sigil stamped on the front."
+	icon = 'icons/obj/custom_items/halstere_clothing.dmi'
+	icon_state = "halstere_card"
+	w_class = 2
+	var/flipped = FALSE
+
+/obj/item/fluff/halstere_card/attack_self(mob/user as mob)
+	flipped = !flipped
+	update_icon()
```
Comment (Lohikar) [inline code/modules/customitems/item_defines.dm]: Almost want to say this should be `queue_icon_update()` so Z-mimic works properly.

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -2265,4 +2265,46 @@ obj/item/clothing/suit/storage/hooded/fluff/make_poncho //Raincoat Poncho - M.A.
...
+/obj/item/fluff/halstere_card/update_icon()
+	if(flipped)
+		icon_state = "halstere_card_open"
+	else
+		icon_state = initial(icon_state)
+
+/obj/item/fluff/halstere_card/examine(mob/user)
+	if(..(user, 1))
+		if(flipped)
+			to_chat(user, "The name 'Halstere, Kalren C.' is stamped on it. An expiration date is listed on it, '2465JAN01'. A pay grade is listed beside the name. 'MAJ/O4'. A number is listed under the expiration date: '14015236810'.")
```
Comment (Lohikar) [inline code/modules/customitems/item_defines.dm]: There has to be a better way to do this.

**Pull request merged**